mozilla/blurts-server
# MNTOR-1859 - expose stats API, starting with scans

# References: 
Jira: MNTOR-1859
Figma: 




# Description

Basic stats API, requires its own bearer token. We could use this for other stats in the future potentially, but start with counts of:

- monthly quota
- current scan count

# How to test

1. ensure `STATS_TOKEN` and `MONTHLY_QUOTA` env vars are set
2. send request to API, example:

```
curl -X GET -H "Content-Type: application/json" -H "Authorization: Bearer ${STATS_TOKEN}" http://localhost:6060/api/v1/stats
```

Expected result:

HTTP status code 200 with output
```
{"success":true,"message":{"monthlyQuota":"17000","scansCount":"0"}}
```

The `scansCount` should reflect the number of rows in the `onerep_scans` table. If the token is invalid, the HTTP status code will be `401` and output

```
{"success":"false"}
```

# Checklist (Definition of Done)
